tools/grift/grpc_plugins_generator.patch

```diff
@@ -0,0 +1,2417 @@
+From 0894590b5020c38106d4ebb2291994668c64f9dd Mon Sep 17 00:00:00 2001
```

Comment:
That said, I would suggest using the patch only when you are making incremental changes to essential code in the underlying third-party repo. When you are basically creating entirely new content (e.g., your tutorial code vs the thrift tutorial code), you should just make them new files in our repo, in a hierarchy under `tools/grift`